src/strategies/math/README.md

```diff
@@ -17,6 +17,8 @@ Currently supported operations are:
 | `a-if-lte-b`  | 3             | (x, a, b) = x <= b ? a : x                 |
 | `a-if-gt-b`   | 3             | (x, a, b) = x > b ? a : x                  |
 | `a-if-gte-b`  | 3             | (x, a, b) = x >= b ? a : x                 |
+| `sum`         | 2             | x + a                                      |
```

Comment:
Hi @anlia260 But using two strategies in a space will already return a sum, is there a reason why we need this?

Reply:
sure, there is some reasons
1. using two strategies in a space will already return a sum is not too obvious ( detailed descriptions cannot be found in the documentation)
2. multiple strategies is less than eight, the math.sum is reduce sum that means multiple strategies expand in the feature ( but now just two)

Reply:
Got you worried, I will remove the sum now, waiting for me.